src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPathContextProvider2.cs

```diff
@@ -42,5 +44,20 @@ public interface IVsPathContextProvider2 : IVsPathContextProvider
         /// <code>InvalidOperationException</code> when it fails to create a context and return appropriate error message.
         /// </throws>
         bool TryCreateSolutionContext(string solutionDirectory, out IVsPathContext2 context);
+
+        /// <summary>
+        /// Attempts to create an instance of <see cref="IVsPathContext"/> containing only the user wide and machine wide configurations.
+        /// If a solution is loaded, note that the values in the path context might not be the actual effective values for the solution.
+        /// If a customer has overriden the `globalPackagesFolder` key or cleared the `fallbackPackageFolders`, these values will be incorrect.
+        /// It is important to keep this scenario in mind when working with this path. To predict differences you can call this in combination with <see cref="TryCreateSolutionContext(out IVsPathContext2)"/>.
+        /// </summary>
+        /// <returns>
+        /// <code>True</code> if operation has succeeded and context was created.
+        /// <code>False</code> otherwise.
+        /// </returns>
+        /// <throws>
+        /// <code>InvalidOperationException</code> when it fails to create a context and return appropriate error message.
+        /// </throws>
```

Comment:
There is no `<throws>` xml doc element documented, but there is an [`<exception>`](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/exception). Did you hover the mouse over the method name to see if IntelliSense shows the docs?

Reply:
I'll try it out.

Reply:
fixed.